### DISCUSSION TITLE
How to tune when printing

### DISCUSSION THREAD

**Discussion by hondaman74** [mriscoc/Ender3V2S1#312] [General]
In other firware I have had loaded I can tune it while printing etc.

I can not figure out how to get to that menu as when I am printing with your firmware it shows the home screen.  I have loaded latest release with the colorful icon set.

Thanks in advance I love the firmware its working great for me
Keith

**Comment by mriscoc** [comment]
If you are printing from Octoprint please read the Wiki: https://github.com/mriscoc/Ender3V2S1/wiki/Octoprint

**Reply by hondaman74** [reply]
yes I am using Octoprint.  I will review thanks

**Reply by MarkforU** [reply]
Only way I've found to get to the Tune Pause and Stop menu buttons is by bringing up a change filament in the display menu, after that you get get to tuning options. I've read the page, enabling the Gcode in Cura works fine to give you time left display. If there's any other way, I wish someone else has idea, I don't like the filament change before print starts or in middle of print to be able to tune. I'm running Octoprint through Python through 2 different PCs to 2 different printers both running Profesional firmware and I have the latest V2 Screens on both printers.

**Reply by mriscoc** [reply]
Check the link
